<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>LED OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1_r">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Red </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1_g">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1_b">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_switch">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>switch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_led">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>now :</string>
    </property>
   </widget>
   <widget class="QDial" name="dial_led">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sound">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Sound ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_song1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_song2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_song3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_song4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QDial" name="dial_sound">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>50</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_sound">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>efect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>298</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
